index.bs

```diff
@@ -569,13 +569,22 @@ Within protected samples, the following constraints apply:
 - [=Temporal Delimiter OBUs=], [=Sequence Header OBUs=], and [=Frame Header OBUs=] (including within a [=Frame OBU=]), [=Redundant Frame Header OBUs=] and [=Padding OBUs=] SHALL be unprotected.
 - [=Metadata OBUs=] MAY be protected.
 - [=Tile Group OBUs=] and [=Frame OBUs=] SHALL be protected. Within [=Tile Group OBUs=] or [=Frame OBUs=], the following applies:
-    - [=BytesOfProtectedData=] SHALL be a multiple of 16 bytes.
-    - [=BytesOfProtectedData=] SHALL end on the last byte of the <code>decode_tile</code> structure (including any trailing bits).
-    - [=BytesOfProtectedData=] SHALL span all complete 16-byte blocks of the <code>decode_tile</code> structure (including any trailing bits).
+	- for the protected scheme <code>[=cbcs=]</code>:
+		- [=BytesOfProtectedData=] SHALL start on the first byte and end on the last byte of the <code>decode_tile</code> structure (including any trailing bits).
```

Comment:
does it mean cbcs also drops the requirement of whole 16-byte block encryption?

Reply:
In this proposal, yes. This is to make it consistent with how the encryption of AVC/HEVC works. However, in practice that does not change anything. Even if the last partial block is included in the protected range, the cbcs scheme indicates that this partial block would remain clear anyway.